wazimap_ng/config/qcluster.py

```diff
@@ -16,5 +16,5 @@ class QCluster:
     Q_CLUSTER = {
         "redis": os.environ.get("REDIS_URL"),
         "workers": int(os.environ.get("Q_CLUSTER_WORKERS", 4)),
-        "recycle": int(os.environ.get("Q_CLUSTER_RECYCLE", 10)),
+        "recycle": int(os.environ.get("Q_CLUSTER_RECYCLE", 1)),
```

Comment:
I think we should rather set this using config, since it's already available as a variable
https://github.com/OpenUpSA/ansible-config/blob/master/apps/wazimap-ng/playbook.yml#L59

Reply:
https://github.com/OpenUpSA/ansible-config/pull/92/files

running it on staging looks like this:
```
 (wazi-worker-recycle) $ PASSWORD_STORE_DIR=~/.pass/openup ansible-playbook --inventory inventory/staging.yml --limit wazimapng1-aws.openup.org.za --tags app apps/wazimap-ng/playbook.yml 
[WARNING]: Invalid characters were found in group names but not replaced, use -vvvv to see details

PLAY [wazimap-ng] ***************************************************************************************************************************************************************************************************************************

TASK [Gathering Facts] **********************************************************************************************************************************************************************************************************************
ok: [wazimapng1-aws.openup.org.za]

TASK [Dokku app exists] *********************************************************************************************************************************************************************************************************************
ok: [wazimapng1-aws.openup.org.za]

TASK [Install RDS CA Cert] ******************************************************************************************************************************************************************************************************************
ok: [wazimapng1-aws.openup.org.za]

TASK [Mount RDS Certificate Authority Certificate where postgres expects it - deploy phase] *************************************************************************************************************************************************
ok: [wazimapng1-aws.openup.org.za]

TASK [Mount RDS Certificate Authority Certificate where postgres expects it - run phase] ****************************************************************************************************************************************************
ok: [wazimapng1-aws.openup.org.za]

TASK [Dokku app config is set] **************************************************************************************************************************************************************************************************************
changed: [wazimapng1-aws.openup.org.za]

TASK [Dokku app domains are configured] *****************************************************************************************************************************************************************************************************
ok: [wazimapng1-aws.openup.org.za]

TASK [Ensure default port mapping doesn't exist] ********************************************************************************************************************************************************************************************
ok: [wazimapng1-aws.openup.org.za]

TASK [Dokku proxy ports are configured] *****************************************************************************************************************************************************************************************************
ok: [wazimapng1-aws.openup.org.za]

TASK [App nginx directory exists] ***********************************************************************************************************************************************************************************************************
ok: [wazimapng1-aws.openup.org.za]

TASK [Upload file size is configured] *******************************************************************************************************************************************************************************************************
ok: [wazimapng1-aws.openup.org.za]

TASK [Nginx is reloaded] ********************************************************************************************************************************************************************************************************************
skipping: [wazimapng1-aws.openup.org.za]

TASK [Your dokku git remote] ****************************************************************************************************************************************************************************************************************
ok: [wazimapng1-aws.openup.org.za] => {
    "msg": "dokku@wazimapng1-aws.openup.org.za:wazimap-ng-staging"
}

PLAY RECAP **********************************************************************************************************************************************************************************************************************************
wazimapng1-aws.openup.org.za : ok=12   changed=1    unreachable=0    failed=0    skipped=1    rescued=0    ignored=0   
```

Reply:
https://wazimap.atlassian.net/browse/WNCM-319?atlOrigin=eyJpIjoiYTVmNjlkYzc5ZDM4NDZiZTgyNzU0OTdhOTVjNzg5MWMiLCJwIjoiaiJ9

Reply:
@jbothma If I merge in this PR will it be affected on the next deployment to staging for testing?